autoload/slime.vim

```diff
@@ -182,6 +182,7 @@ function! s:TmuxSend(config, text)
     call s:WritePasteFile(chunk)
     call s:TmuxCommand(a:config, "load-buffer " . g:slime_paste_file)
     if bracketed_paste
+      call s:TmuxCommand(a:config, "send-keys -X -t " . shellescape(a:config["target_pane"]) . " cancel")
```

Comment:
why is it inside the `if bracketed_paste` branch? 🤔

Reply:
When `bracketed_paste` is off, the side effect of `cancel` is to bring focus to the bottom, but it does not affect paste. It is more a personal preference to send a `cancel` or not.

I intend not to mess with other people's workflow if they don't complain about it.

Reply:
Can you update the PR to put the `cancel` just above the `if`?

I've experimented and it doesn't seem like a bad workflow.

If there's pushback, we can add a flag to make this conditional. (something like `g:slime_send_cancel` ?)

Reply:
The requested update is done.